packages/wdio-cli/src/launcher.js

```diff
@@ -241,10 +244,12 @@ class Launcher {
      * Start instance in a child process.
      * @param  {Array} specs  Specs to run
      * @param  {Number} cid  Capabilities ID
+     * @param  {Number} rid  Runner ID override
+     * @param  {Number} retries  Number of retries remaining
      */
-    startInstance (specs, caps, cid, server) {
+    startInstance (specs, caps, cid, server, rid, retries) {
         let config = this.configParser.getConfig()
-        cid = this.getRunnerId(cid)
+        cid = rid || this.getRunnerId(cid)
```

Comment:
Can we add a comment saying that in order to retry tests with the same `cid` we pass the original `cid` along as `rid` to be able to identify tests better